boards/CMakeLists.txt

```diff
@@ -356,13 +354,13 @@ function(create_arm_binary_or_tests_for_board
     set(APP_CANMSGS_HEADER_FILE_NAME "App_CanMsgs.h")
 
     set(BOARD_SPECIFIC_AUTOGENERATED_APP_SRC_DIR
-            "${CMAKE_CURRENT_SOURCE_DIR}/Src/App/auto_generated")
+            "${CMAKE_CURRENT_BINARY_DIR}/Src/App/auto_generated")
```

Comment:
Wait, didn't you want to get rid of auto_generated here as well?

Reply:
Oh sorry no, but yeah, it would make sense to. Removed.